packages/components/src/unit-control/index.tsx

```diff
@@ -56,6 +57,15 @@ function UnforwardedUnitControl(
 	}: WordPressComponentProps< UnitControlProps, 'input', false >,
 	forwardedRef: ForwardedRef< any >
 ) {
+	if ( typeof unitProp !== 'undefined' ) {
+		deprecated( 'UnitControl unit prop', {
+			since: '5.6',
+			hint:
+				'Please provide a unit with a value through the `value` prop.',
```

Comment:
Pretty small potatoes but this might be a tiny bit more easily read as something like:
```suggestion
				'The unit should be provided within the `value` prop.',
```

Reply:
Makes sense! Sometimes my written english isn't the most fluent, thank you!